pkg/virt-api/rest/subresource.go

```diff
@@ -61,7 +61,7 @@ func (app *SubresourceAPIApp) streamRequestHandler(request *restful.Request, res
 
 	vmi, code, err := app.fetchVirtualMachineInstance(vmiName, namespace)
 	if err != nil {
-		log.Log.Reason(err).Error("Failed to gather remote exec info for subresource request.")
+		log.Log.Object(vmi).Reason(err).Error("Failed to gather remote exec info for subresource request.")
```

Comment:
vmi is probably null here, you'll have to report the vmiName and namespace in the log instead i think.

Reply:
yep, you are right, or I can create dummy VMI

Reply:
yup, either way works for me